app/models/ems_refresh/save_inventory.rb

```diff
@@ -116,7 +116,7 @@ def save_vms_inventory(ems, hashes, target = nil)
 
         found.save!
         h[:id] = found.id
-        found.send(:clear_association_cache)
+        found.reload
```

Comment:
> The problem is that clear_association_cache allows the associations and their associations to be GC'd but doesn't allow the current object's attributes to be GC'd.

That's pretty much exactly what we want, right?  Or more specifically, reload will go refetch the attributes, so regardless we would have an object with the attributes inside.  The associations were always the heavy hitter as far as memory goes, which is why we wanted to use clear_association_cache.